### PR TITLE
Fix delete images in bulk test && small improvements

### LINKED ISSUE #1650: Fix AdminEnhancedMediaGalleryDeleteImagesInBulkTest
AC:
- unskip test
- test must pass

### LINKED ISSUE #1652: Incorrent bulk delete confirmation button name
### Preconditions (*)
Install Magento + AdobeIMS + Adobe Stock Integration
Enable Enhanced Media Gallery

### Steps to reproduce (*)
1. Open Content -> Media Gallery
2. Upload several images
3. Click "Delete Images..."
4. Select several images
5. Click Delete and see the confirmation popup

### Expected result (*)
Let's rename the button to just "Delete"

### Actual result (*)
"Delete Image" (singular) button name on the confirmation popup

image

### LINKED ISSUE #1654: Backend error message is ignored on file upload
### Description

`app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image-uploader.js` is ignoring the errors passed from backend and always displays a single `Cannot upload "imagename". This file format is not supported.`

### Steps to reproduce (*)
1. Open `Content -> Media Gallery`
2. Create a new directory
3. Delete newly created directory from filesystem
4. Upload an image to the created directory (that is still displayed on frontend)

### Expected result (*)
`Directory <directoryname> does not exist in media directory.` error is displayed

### Actual result (*)
`Cannot upload "imagename". This file format is not supported.` error is displayed

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1650: Fix AdminEnhancedMediaGalleryDeleteImagesInBulkTest
2. Fixes magento/adobe-stock-integration#1654: Backend error message is ignored on file upload
3. Fixes magento/adobe-stock-integration#1652: Incorrent bulk delete confirmation button name
4. Handle image uploading only for new Media Gallery

### Manual testing scenarios (*)

1. ...
2. ...

### CODE REVIEW

**Comment by Nazar65** [comment]
@magento run all tests

File: MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryAssertMassActionModeNotActiveActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminEnhancedMediaGalleryAssertMassActionModeNotActiveActionGroup">
+        <annotations>
+            <description>Asserts that massaction mode is terminated</description>
+        </annotations>
+       
+
+        <dontSeeElement selector="{{AdminEnhancedMediaGalleryMassActionSection.totalSelected}}" stepKey="verifyTeminateMAssAction"/>
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryAssertMassActionModeNotActiveActionGroup.xml]: :)
```suggestion
        <dontSeeElement selector="{{AdminEnhancedMediaGalleryMassActionSection.totalSelected}}" stepKey="verifyTeminateMassAction"/>
```

Comment (Nazar65) [reply MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryAssertMassActionModeNotActiveActionGroup.xml]: :+1:

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**